Core/TokenServices/TokenGenerationServices/TokenGeneration.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Data.AppDbContext;
+using Data.Models;
+
+
+namespace Core.TokenServices.TokenGenerationService
+{
+    public class TokenGenerationService : ITokenGenerationService
+    {
+        private readonly TeejayDbContext _context;
+        private static readonly Random _random = new Random();
```

Comment:
Please use System.Security.Cryptography.RandomNumberGenerator instead. This is preferred because it is a cryptographically secure random number generator class.